components/aws/sagemaker/batch_transform/component.yaml

```diff
@@ -56,7 +56,7 @@ outputs:
 - {name: output_location, description: S3 URI of the transform job results.}
 implementation:
   container:
-    image: amazon/aws-sagemaker-kfp-components:1.1.1
+    image: amazon/aws-sagemaker-kfp-components:1.1.2
```

Comment:
we will have to do a release before we can change the image here 
could you please revert it

Reply:
Thanks. I've reverted it.
Could you please let me know how to release the new image?

Reply:
we run some tests and make documentation changes before the release 

this is how you build the image :
```
cd components/aws/sagemaker
docker build -f ./Dockerfile . -t amazon/aws-sagemaker-kfp-components:1.1.2
```

Reply:
Got it. Thank you.